examples/cryptocurrency-advanced/backend/build.rs

```diff
@@ -0,0 +1,14 @@
+extern crate protoc_rust;
+
+use protoc_rust::Customize;
+
+fn main() {
+    protoc_rust::run(protoc_rust::Args {
+        out_dir: "src/proto",
+        input: &["src/proto/cryptocurrency.proto"],
+        includes: &["src/proto", "../../../exonum/src/encoding/protobuf/proto"],
```

Comment:
Somehow, this doesn't seem like a sustainable solution. How is it supposed to work for published crates?

Reply:
They have to get .proto files separately
* maybe there is a way to distribute them with cargo package that I did not find with brief search

Reply:
I think that we just should create exonum proto files registry in the repository root?